### PR TITLE
Sync changes in registries from upstream

### PR DESCRIPTION
**Pull request by amisevsk**
Pulls in fixes from plugin and devfile registries.

Changes should be current as of commits
- https://github.com/eclipse/che-devfile-registry/commit/2d14db69a93c775b3cea835fde96e975c0f2e1a6 in devfile registry
- https://github.com/eclipse/che-plugin-registry/commit/5da013d9a1089550c584c4d1ac4b39ce630f1184 in plugin registry

### CODE REVIEW

**Comment by amisevsk** [comment]
ping @rhopp, I don't know the process for merging.

File: dependencies/che-plugin-registry/build/dockerfiles/rhel.Dockerfile [hunk 1]
```diff
@@ -20,9 +20,7 @@ USER 0
 ################# 
 
 ARG BOOTSTRAP=false
-ENV BOOTSTRAP=${BOOTSTRAP}
```
Comment (nickboldt) [inline dependencies/che-plugin-registry/build/dockerfiles/rhel.Dockerfile]: not sure this works in Brew. Please leave it as is.

File: dependencies/che-devfile-registry/build/dockerfiles/rhel.Dockerfile [hunk 2]
```diff
@@ -65,11 +65,8 @@ WORKDIR /build/
 
 RUN ./check_mandatory_fields.sh devfiles
 RUN ./index.sh > /build/devfiles/index.json
-RUN chmod -R g+rwX /build/devfiles
-
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
-# Does not work in brew; need to run this online, cache in tarball and add to Brew
-# RUN ./cache_projects.sh devfiles resources && chmod -R g+rwX /build
```
Comment (nickboldt) [inline dependencies/che-devfile-registry/build/dockerfiles/rhel.Dockerfile]: please also add commented `./cache_images.sh devfiles resources` here as in non-RHEL version.

File: dependencies/che-devfile-registry/build/dockerfiles/rhel.Dockerfile [hunk 3]
```diff
@@ -65,11 +65,8 @@ WORKDIR /build/
 
 RUN ./check_mandatory_fields.sh devfiles
 RUN ./index.sh > /build/devfiles/index.json
-RUN chmod -R g+rwX /build/devfiles
-
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
-# Does not work in brew; need to run this online, cache in tarball and add to Brew
```
[the remark this answers is not in the document]
Comment (amisevsk) [reply dependencies/che-devfile-registry/build/dockerfiles/rhel.Dockerfile]: Done -- although the commented out section in this file does not match where it's located in upstream (caching is only done in offline build). I've added the section to that part of the dockerfile.

File: dependencies/che-devfile-registry/build/dockerfiles/rhel.Dockerfile (hunk 2 above)
Comment (amisevsk) [reply dependencies/che-devfile-registry/build/dockerfiles/rhel.Dockerfile]: :+1:

File: dependencies/che-plugin-registry/build/dockerfiles/rhel.Dockerfile (hunk 1 above)
Comment (amisevsk) [reply dependencies/che-plugin-registry/build/dockerfiles/rhel.Dockerfile]: :+1: done

**Pull request merged**